types/src/front/lib/actions/registry.ts

```diff
@@ -149,6 +149,36 @@ export const DustProdActionRegistry = createActionRegistry({
       },
     },
   },
+  "assistant-builder-instructions-suggestions": {
+    app: {
+      workspaceId: PRODUCTION_DUST_APPS_WORKSPACE_ID,
+      appId: "d995d868a8",
+      appHash:
+        "174d83511a51ba35dd57e840a2189a75474695e87b75902c9d04955367804446",
```

Comment:
Sure ! Honestly this was a quick first shot. Plenty of room for experimentation.
Will try the quick wins you describe, cc @Duncid do you have material also?

No need to wait to ship though, since it's gated